CondFormats/PCLConfig/interface/AlignPCLThresholds.h

```diff
@@ -0,0 +1,58 @@
+#ifndef _AlignPCLThresholds_h_
+#define _AlignPCLThresholds_h_
+
+#include "CondFormats/PCLConfig/interface/AlignPCLThreshold.h"
+#include "CondFormats/Serialization/interface/Serializable.h"
+
+#include <map>
+#include <string>
+#include <vector>
+
+using namespace std;
+
+class AlignPCLThresholds{
+ public:
+  typedef map<string,AlignPCLThreshold> threshold_map;
+  enum coordType {X, Y, Z, theta_X, theta_Y, theta_Z, extra_DOF, endOfTypes}; 
+
+  AlignPCLThresholds(){}
+  virtual ~AlignPCLThresholds(){}
+
+  void setAlignPCLThreshold(string AlignableId, const AlignPCLThreshold &Threshold);
```

Comment:
please pass `AlignableId` as `const std::string&`
the same applies in the other methods